app/assets/javascripts/surveys.js

```diff
@@ -140,6 +140,15 @@
     }
   }
 
+  var tlsSurveyUrl = function () {
+    if (window.matchMedia( "(max-width: 800px)" ).matches) {
```

Comment:
Is there any chance that there is something already on the page that can tell us if something is / isn't mobile?  Wary of having this `max-width: 800px` conditional get out of date with conditionals we use for real CSS styling.

Reply:
Actually, from chatting to #frontend folks it seems there isn't anything in our JS bundle that does this already, so this is probably fine.  It's also likely to be temporary as eventually all our surveys should on mobile.  Let's leave this as-is.